specification/containerstorage/resource-manager/readme.md

```diff
@@ -0,0 +1,65 @@
+# containerstorage
+
+> see https://aka.ms/autorest
+This is the AutoRest configuration file for containerstorage.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+To see additional help and options, run:
+
+> `autorest --help`
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the containerstorage.
+
+```yaml
+openapi-type: arm
+tag: package-2023-03-01-preview
+```
+
+### Tag: package-2023-03-01-preview
+
+These settings apply only when `--tag=package-2023-03-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-03-01-preview'
+input-file:
+  - Microsoft.ContainerStorage/preview/2023-03-01-preview/containerstorage.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-python
```

Comment:
We need to remove the `azure-sdk-for-python-track2`, because we no longer have that

Reply:
let us come back to this later, since no pipeline is complaining about this